inc/datahandlers/user.php

```diff
@@ -294,23 +294,10 @@ function verify_website()
 	{
 		$website = &$this->data['website'];
 
-		if(empty($website) || my_strtolower($website) == 'http://' || my_strtolower($website) == 'https://')
-		{
-			$website = '';
-			return true;
-		}
-
-		// Does the website start with http(s)://?
-		if(my_strtolower(substr($website, 0, 4)) != "http")
-		{
-			// Website does not start with http://, let's see if the user forgot.
-			$website = "http://".$website;
-		}
-
-		if(!filter_var($website, FILTER_VALIDATE_URL))
+		if($website && !filter_var($website, FILTER_VALIDATE_URL))
```

Comment:
And this is quite a big change - earlier URLs without protocol like `www.something.com` could be entered by less advanced users + not sure why you changed `false` to `true`.